guides/tutorials/agent-config.md

```diff
@@ -190,6 +194,12 @@ a JSON object like the
 
 Stringified JSON object of the transaction.
 
+### `indyLedgers.transactionAuthorAgreement`
+
+**Type**: `TransactionAuthorAgreement`
+
+JSON representation specifying the version and acceptance mechanism. The version is the unique version of the transaction author agreement acceptance mechanism list (AML). The acceptance mechanism refers to the acceptance mechanism label of the item in the AML. For more details you may consult the [indy-node docs on AML](https://github.com/hyperledger/indy-node/blob/master/docs/source/transactions.md#transaction_author_agreement_aml)
```

Comment:
Its a good description, but I ment more in the style of like the walletConfig where the keys are seperately explained.

Like here: 

https://animo.github.io/aries-javascript-docs/guides/tutorials/agent-config#walletconfigid